lib/livebook/hubs/team.ex

```diff
@@ -2,18 +2,36 @@ defmodule Livebook.Hubs.Team do
   use Ecto.Schema
   import Ecto.Changeset
 
+  alias Livebook.Teams
+  alias Livebook.Hubs.TeamClient
+
+  def get_deployment_groups(team) do
+    TeamClient.get_deployment_groups(team.id)
+  end
+
+  def create_deployment_group(team, deployment_group),
+    do: Teams.create_deployment_group(team, deployment_group)
+
+  def update_deployment_group(team, deployment_group),
+    do: Teams.update_deployment_group(team, deployment_group)
+
+  def delete_deployment_group(team, deployment_group),
+    do: Teams.delete_deployment_group(team, deployment_group)
```

Comment:
Sorry, we should use all of these four functions directly from `Livebook.Teams`, we can skip the proxy here. I know it is a bit confusing all of the modules and their placement.